### PR TITLE
Step2accordian 2

### PR DESCRIPTION
**Pull request by pmbohm**

### CODE REVIEW

**Comment by anguss00** [comment]
I'm taking this on

**Comment by danfruehauf** [comment]
@anguss00 @pmbohm Please don't commit anything on that, I want to clean this PR so commits can be squashed...

**Comment by anguss00** [comment]
No problemo @danfruehauf

File: web-app/js/portal/details/SpatialSubsetControlsPanel.js [hunk 1]
```diff
@@ -40,43 +38,52 @@ Portal.details.SpatialSubsetControlsPanel = Ext.extend(Ext.Panel, {
...
-
     },
 
     _addSpatialConstraintDisplayPanel: function() {
         this.spatialConstraintDisplayPanel = new Portal.details.SpatialConstraintDisplayPanel({
             map: this.map,
-            width: this.width
+            cls: "floatLeft",
+            unstyled: true,
+            width: 160
```
Comment (anguss00) [inline web-app/js/portal/details/SpatialSubsetControlsPanel.js]: what do you think about having these widths and spacer heights set as constants here @pmbohm? Although the only issue with that is there are a fair few different numbers here for the specific filter panels

Comment (anguss00) [inline web-app/js/portal/details/SpatialSubsetControlsPanel.js]: Or they could be even defined at the filter panel object level if they are consistent for each filter panel type

File: web-app/js/portal/details/SubsetItemsTabPanel.js [hunk 2]
```diff
@@ -7,42 +7,35 @@
...
 
     constructor: function (cfg) {
-        var childPanelConfig =  { map: cfg.map, layer: cfg.layer, mapPanel: cfg.mapPanel };
+        var childPanelConfig =  { map: cfg.map, layer: cfg.layer };
 
         this.subsetPanel = new Portal.details.SubsetPanel(childPanelConfig);
         this.infoPanel = new Portal.details.InfoPanel(childPanelConfig);
         this.stylePanel = new Portal.details.StylePanel(childPanelConfig);
-        this.mapOptionsPanel = new Portal.ui.MapOptionsPanel(childPanelConfig);
+        //this.mapOptionsPanel = new Portal.ui.MapOptionsPanel(childPanelConfig + mappanel);
```
Comment (anguss00) [inline web-app/js/portal/details/SubsetItemsTabPanel.js]: commented out line here

File: web-app/js/portal/details/SpatialSubsetControlsPanel.js (hunk 1 above)
Comment (pmbohm) [inline web-app/js/portal/details/SpatialSubsetControlsPanel.js]: These values are specific to the layout of this component and are not referenced elsewhere or more than once each, so i dont think there is anything to gain at this stage

**Pull request closed without merging**